app/2.0/start/first-element/step-4.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Step 4: React to input"
+subtitle: "Build your first Polymer element"
+---
+
+Of course, a button isn't a button if you can't click it. 
+
+To toggle the button, we will add an event listener. To make event listener methods available, we will include the methods we need by importing and using the `Polymer.GestureEventListeners` mixin. 
```

Comment:
2nd sentence here is wrong. The mixin adds support for gesture events that work the same way across different browsers. (You should also probably introduce the concept of mixins somewhere in this step.)

At some point, you might say something like, GestureEventListeners provides a `tap` event that works like the standard `click` event on desktop browsers, and like a native tap event on mobile browsers.

Maybe @azakus could tell us whether this is really still necessary...